app/controllers/artists_controller.rb

```diff
@@ -1,16 +1,33 @@
 class ArtistsController < ApplicationController
+  include VideosHelper
+  include ActionController::MimeResponds
   before_action :require_admin, except: [:index, :show]
   before_action :set_artist, only: [:show, :update, :edit, :destroy]
 
+
+
+  #One of of the changes I made to fix the some of the errors was on line 12 to change @artists to Artist.
+
   def index
-    @artists = Artist.paginate(page: params[:page], per_page: 4)
+    @artists = Artist.paginate(page: params[:page], per_page: 20)
+    @filterrific = initialize_filterrific(
+      Artist,
+      params[:filterrific]
+    ) or return
+    @artists = @filterrific.find.page(params[:page])
+ 
+    respond_to do |format|
+      format.html
+      format.js
+    end
   end
 
   def new
     @artist = Artist.new
   end
 
   def show
+    puts @artist
```

Comment:
This probably can be removed now right? Looks like it was for debugging purposes.